homeassistant/components/elmax/config_flow.py

```diff
@@ -170,25 +177,29 @@ async def async_step_panels(
 
     async def async_step_reauth(self, entry_data: Mapping[str, Any]) -> FlowResult:
         """Perform reauth upon an API authentication error."""
-        self._reauth_username = entry_data.get(CONF_ELMAX_USERNAME)
         self._reauth_panelid = entry_data.get(CONF_ELMAX_PANEL_ID)
```

Comment:
```suggestion
        self.entry = self.hass.config_entries.async_get_entry(self.context["entry_id"])
```
I think it would be better to get the entry here and use that further down in the next steps

Reply:
Done. I also had to align some test cases that where not setting up an entry.